compiler_opt/rl/corpus.py

```diff
@@ -238,7 +240,10 @@ def __init__(self,
     output) and validated.
 
     Args:
-      data_path: corpus directory.
+      location: either the path to the corpus description json file in a corpus
```

Comment:
Is it better that we only allow the input being the corpus_description path? The only concern is that it adds some migration cost...

Reply:
I think that'd be better, but to avoid migration pain, we could stage it: this patch first; then warn about deprecation; then deprecate 

we can skip the 2nd step because we don't have a release model and so when folks would notice the change is out of our control; but can work with e.g. Fuchsia to update their scripts.

Reply:
the migration plan sgtm!